packages/react-sdk/src/app/custom-components/image.tsx

```diff
@@ -28,12 +28,17 @@ export const Image = forwardRef<ElementRef<typeof defaultTag>, Props>(
       return loaders.localImageLoader();
     }, [asset, params]);
 
-    if (asset === null || loader == null)
-      return <SdkImage {...props} ref={ref} />;
+    let src = props.src;
 
-    return (
-      <WebstudioImage {...props} loader={loader} optimize={true} ref={ref} />
-    );
+    if (asset != null) {
+      src = asset.path;
+    }
+
+    if (asset === null || loader == null) {
```

Comment:
shouldn't loader be either undefined or defined? why do we need to support null for it? I would just make loader optional and that's it, trying to not have to support both null and undefined as much as possible.

We would mostly only need null when using dom API and we can keep those usages local and not expose it to consumers